core/writerassembly.go

```diff
@@ -91,7 +91,6 @@ func (asm *WriterAssembly) Write(messages []*Message) {
 	contentLen := 0
 	for _, msg := range messages {
 		msgCopy := msg.Clone()
```

Comment:
Just for reference (we already talked about this): Until #81 has been implemented drop is broken here as messages does not contain the "originals" any more.